service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProvider.java

```diff
@@ -91,28 +97,21 @@ private String getHearingBooking(CaseData data) {
         return ordersAndDirectionsBuilder.build();
     }
 
-    @SuppressWarnings("unchecked")
-    private void appendOrders(Map<String, Object> orders, ImmutableList.Builder<String> builder) {
-        Optional.ofNullable(orders.get("orderType")).ifPresent(orderTypes -> {
-            for (String typeString : (List<String>) orderTypes) {
-                builder.add(OrderType.valueOf(typeString).getLabel());
-            }
-        });
+    private void appendOrders(Orders orders, ImmutableList.Builder<String> builder) {
+        defaultIfNull(orders.getOrderType(), Collections.<OrderType>emptyList()).stream()
```

Comment:
```suggestion
        defaultIfNull(orders.getOrderType(), emptyList()).stream()
```

I don't think you need to be verbose with type here

Reply:
@swalker125 it wont compile without explicit type information, unless you do
      List<OrderType> types=  defaultIfNull(orders.getOrderType(), emptyList());

        types.stream()
            .map(OrderType::getLabel)
            .forEach(builder::add);